### ISSUE TITLE
An appropriate license

### ISSUE THREAD

**Issue by rossbar** [BIDS-numpy/numpy-handbook#1]
What would be the best license for the project? Looking at other open "technical" books, it seems like some combination of CC-4.0 for the the book part and BSD-3 for any associated code would be best. Perhaps there are nuances related to code being embedded directly within the text (e.g. like Jupyter notebooks)?

@stefanv do you have any insight from Elegant Scipy?

**Comment by stefanv** [comment]
Yes, I would say CC-BY for text, CC0 for data, BSD for code. That allows very wide reuse, while still crediting the authors.

**Comment by rossbar** [comment]
Great, thanks for the insight!
